source/.res/replace.txt

```diff
@@ -3,7 +3,7 @@
 .. |mongodb-ce| replace:: MongoDB 4.0 Community Edition
 .. |mongodb| replace:: MongoDB
 .. |percona| replace:: Percona
-.. |prev-version| replace:: 4.0
+.. |prev-version| replace:: 3.6
```

Comment:
I'm not 100% sure that I know what this does, but are you sure 4.0 should be replaced with 3.6 here - notice you are mergeing to 4.0 branch?

Reply:
There was a bug with previous version variable (from 4.0 to 4.0. I fixed that.

Reply:
Cool! Thanks!